addons/mail/static/src/models/notification_group/notification_group.js

```diff
@@ -101,6 +101,12 @@ function factory(dependencies) {
         id: attr({
             required: true,
         }),
+        /**
+         * Determines the position of the group inside the list.
+         */
+        sequence: attr({
+            default: 0,
+        }),
```

Comment:
Alphabetical, and also give more info on how position is determined by this. Also make explanation as context-free as possible: current way this attribute is declared gives too many questions:

- What list are you talking about? What's the deal with positioning in this list?
- What are the other items in this list?
- What does this number represents more concretely?
- etc.

Answers are obvious for you and me right now, but it's not for other developers right now. Even for us we may forget details in the future, and it's a problem not having explicit and bounded declaration of this field because we are prone to making big mistakes.